### PR TITLE
vimPlugins.presenting-vim: init at 2018-11-25

### PR DESCRIPTION
**Pull request by kalbasit**
###### Motivation for this change

https://github.com/sotte/presenting.vim

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/vim-plugins/generated.nix [hunk 1]
```diff
@@ -135,18 +135,6 @@ let
...
-    version = "2020-08-09";
-    src = fetchFromGitHub {
-      owner = "prabirshrestha";
-      repo = "asyncomplete.vim";
-      rev = "0d4ec1805d50fd39b8b41626e0761736feb307d0";
-      sha256 = "0lcqdh51fd8lx1g02dcwrfnnx3zfm71g57nv2pmin5ap7qgh8kir";
-    };
-    meta.homepage = "https://github.com/prabirshrestha/asyncomplete.vim/";
-  };
-
```
Comment (NieDzejkob) [inline pkgs/misc/vim-plugins/generated.nix:149]: What is the reasoning behind removing this package?

Comment (trepetti) [reply pkgs/misc/vim-plugins/generated.nix:149]: This was done by `update.py`. It looks like the package has a new owner on GitHub, and the `update.py` script follows redirect links. A recent PR I made for vimPlugins featured the same update.

Comment (kalbasit) [reply pkgs/misc/vim-plugins/generated.nix:149]: Hmm the package has not moved from https://github.com/prabirshrestha/asyncomplete.vim, and I did not make this change. I'll re-run the script :crossed_fingers:

Comment (kalbasit) [reply pkgs/misc/vim-plugins/generated.nix:149]: Looks like it was introduced in 2bc02f29256c0ba5c366330405502637c73c1fd1 but was not introduced properly.

Comment (trepetti) [reply pkgs/misc/vim-plugins/generated.nix:149]: Yes that looks to be the issue. Good eye. It was not included in vim-plugin-names.

Comment (kalbasit) [reply pkgs/misc/vim-plugins/generated.nix]: I addressed this issue, PTAL.

**Pull request closed without merging**